src/Domain/Prices/JsonApi/V1/PriceResource.php

```diff
@@ -48,17 +49,20 @@ public function attributes($request): iterable
                 'decimal' => $comparePrice?->decimal,
                 'value' => $comparePrice?->value,
             ],
+            ...ResourceManifest::for(static::class)->attributes()->toResourceArray($this),
         ];
     }
 
     /**
      * Get the resource's relationships.
      *
-     * @param \Illuminate\Http\Request|null $request
+     * @param  \Illuminate\Http\Request|null  $request
      * @return iterable
      */
     public function relationships($request): iterable
     {
-        return [];
+        return [
+            ...ResourceManifest::for(static::class)->relationships()->toResourceArray($this),
```

Comment:
<img width="1333" alt="image" src="https://user-images.githubusercontent.com/5587309/211639916-2c66fd92-4886-4f76-9446-78eee2cc0462.png">

Tady kdyz se to odvolava na ten interface, tak to nezna tu metodu. Nejaky napad, jak to uhladit? Treba jen definovat spolecnou `toArray()`, ale nevim, co s tema parametrama.